SnailTests/ObservableTests.swift

```diff
@@ -260,47 +260,47 @@ class ObservableTests: XCTestCase {
         observable.on(.next("1"))
 
         waitForExpectations(timeout: 1) { _ in
-            XCTAssert(received.count == 1)
-            XCTAssert(received.first == "3")
+            XCTAssertEqual(received.count, 1)
+            XCTAssertEqual(received.first, "3")
         }
     }
 
     func testSkipFirst() {
         let observable = Observable<String>()
         var received: [String] = []
 
-        observable.skip(first: 2).subscribe(onNext: { string in
-            received.append(string)
-
-            XCTAssert(received.count == 1)
-            XCTAssert(received.first == "3")
-        })
+        observable.skip(first: 2).subscribe(onNext: { received.append($0) })
 
         observable.on(.next("1"))
         observable.on(.next("2"))
         observable.on(.next("3"))
+        XCTAssertEqual(received.count, 1)
+        XCTAssertEqual(received.first, "3")
     }
 
     func testSkipError() {
         let observable = Observable<String>()
 
-        var error: Error?
+        var error: TestError?
+        observable.skip(first: 2).subscribe(onError: { error = $0 as? TestError })
+        observable.on(.error(TestError.test))
 
-        observable.skip(first: 2).subscribe(onError: {
-            error = $0
-        })
+        XCTAssertEqual(error, .test)
+    }
 
-        observable.on(.error(TestError.test))
+    func testSkipDone() {
+        let observable = Observable<String>()
+        var done = false
 
-        XCTAssertNotNil(error)
-        XCTAssert(error is TestError)
+        observable.skip(first: 2).subscribe(onDone: { done = true })
+        observable.on(.done)
+
+        XCTAssertEqual(done, true)
```

Comment:
@russellbstephens I think this test was still missing after adding `testSkipError`